py-scoring/src/ai/h2o/sparkling/Initializer.py

```diff
@@ -173,10 +173,11 @@ def __add_url_to_classloader(gateway, url):
         logger = Initializer.__get_logger(jvm)
         while loader:
             try:
+                urlClassLoaderClass = jvm.py4j.reflection.ReflectionUtil.classForName("java.net.URLClassLoader")
```

Comment:
Why do we have this in the while-loop?

the instance of urlClassLoaderClass will not change in the loop